deploy/unifi.sh

```diff
@@ -1,30 +1,43 @@
 #!/usr/bin/env sh
 
-#Here is a script to deploy cert to unifi server.
+# Here is a script to deploy cert on a Unifi Controller or Cloud Key device.
+# It supports:
+#   - self-hosted Unifi Controller
+#   - Unifi Cloud Key Gen1
+#   - UnifiOS (Cloud Key Gen2)
```

Comment:
For clarity, I'd adjust documentation to this:
```
#  - Unifi Cloud Key (Gen1/2/2+)
#  - Unifi Cloud Key running UnifiOS (v2.0.0+, Gen2/2+ only)
```

Reply:
Done